src/llmq/quorums.h

```diff
@@ -244,18 +241,25 @@ class CQuorumManager
     const CActiveMasternodeManager* const m_mn_activeman;
     const CMasternodeSync& m_mn_sync;
     const CSporkManager& m_sporkman;
+    const bool m_quorums_watch{false};
+
+private:
```

Comment:
drop

Reply:
We are still using it

https://github.com/dashpay/dash/blob/00cde00d6a8befd05e4d83c1871e9b59d9e03824/src/llmq/quorums.cpp#L256-L258

https://github.com/dashpay/dash/blob/00cde00d6a8befd05e4d83c1871e9b59d9e03824/src/llmq/quorums.cpp#L371-L373

Reply:
I guess:

```suggestion
```
`